Rakefile

```diff
@@ -1,13 +1,15 @@
+# frozen_string_literal: true
```

Comment:
Let's make a separate PR that changes the hash formation away from the rocket. This PR should scope itself just to fixing the JWT ACL paths parameter.